src/main/java/com/codurance/guru/craftspeople/CraftspeopleController.java

```diff
@@ -23,4 +22,18 @@ public Craftsperson retrieveCraftsperson(@PathVariable Integer craftspersonId) {
         return craftspeopleService.retrieveAllCraftsperson();
     }
 
+    @PutMapping("craftspeople/addmentee")
+    String setMentee(@RequestBody Map<String, String> mentorAndMenteesIds) {
```

Comment:
You can define a class with the fields `mentorId` / `menteeId` and use it with `@RequestBody`